src/gui/plugins/view_angle/ViewAngle.cc

```diff
@@ -167,7 +167,7 @@ void ViewAngle::LoadConfig(const tinyxml2::XMLElement *)
   this->dataPtr->moveToModelService = "/gui/move_to/model";
   this->dataPtr->node.Advertise(this->dataPtr->moveToModelService,
       &ViewAngle::OnMoveToModelService, this);
-  ignmsg << "Move to model service on ["
+  gzmsg << "Move to model service on ["
          << this->dataPtr->moveToModelService << "]" << std::endl;
```

Comment:
```suggestion
  gzmsg << "Move to model service on ["
        << this->dataPtr->moveToModelService << "]" << std::endl;
```

Reply:
Here and below

Reply:
[2ea163b](https://github.com/gazebosim/gz-sim/pull/1877/commits/2ea163b7d1d410cc841cafbad0b107b001adc7f4)